composer.json

```diff
@@ -12,7 +12,7 @@
   "require": {
     "php": ">=5.6",
     "ext-zip": "*",
-    "symfony/filesystem": "~2.8",
+    "symfony/filesystem": "~3.2",
```

Comment:
@Quetzacoalt91 Why do you update this package ?

Reply:
That was done for composer, in order to match the required version by `prestashop/php-coding-standards`: https://packagist.org/packages/prestashop/php-coding-standards

Reply:
Ok